modules/exploits/unix/smtp/opensmtpd_mail_from_rce.rb

```diff
@@ -12,47 +12,49 @@ class MetasploitModule < Msf::Exploit::Remote
   include Msf::Exploit::Expect
 
   def initialize(info = {})
-    super(update_info(info,
-      'Name'           => 'OpenSMTPD MAIL FROM Remote Code Execution',
-      'Description'    => %q{
-        This module exploits a command injection in the MAIL FROM field during
-        SMTP interaction with OpenSMTPD to execute a command as the root user.
-      },
-      'Author'         => [
-        'Qualys',                               # Discovery and PoC
-        'wvu',                                  # Module
-        'RageLtMan <rageltman[at]sempervictus>' # Module
-      ],
-      'References'     => [
-        ['CVE', '2020-7247'],
-        ['URL', 'https://seclists.org/oss-sec/2020/q1/40']
-      ],
-      'DisclosureDate' => '2020-01-28',
-      'License'        => MSF_LICENSE,
-      'Platform'       => 'unix',
-      'Arch'           => ARCH_CMD,
-      'Privileged'     => true,
-      'Targets'        => [
-        ['OpenSMTPD < 6.6.1',
-          'MyBadChars' => "!\#$%&'*?`{|}~\r\n".chars
-        ]
-      ],
-      'DefaultTarget'  => 0,
-      'DefaultOptions' => {'PAYLOAD' => 'cmd/unix/reverse_netcat'},
-      'Notes'          => {
-        'Stability'    => [CRASH_SAFE],
-        'Reliability'  => [REPEATABLE_SESSION],
-        'SideEffects'  => [IOC_IN_LOGS]
-      }
-    ))
+    super(
+      update_info(
+        info,
+        'Name' => 'OpenSMTPD MAIL FROM Remote Code Execution',
+        'Description' => %q{
+          This module exploits a command injection in the MAIL FROM field during
+          SMTP interaction with OpenSMTPD to execute a command as the root user.
+        },
+        'Author' => [
+          'Qualys', # Discovery and PoC
+          'wvu', # Module
+          'RageLtMan <rageltman[at]sempervictus>' # Module
+        ],
+        'References' => [
+          ['CVE', '2020-7247'],
+          ['URL', 'https://seclists.org/oss-sec/2020/q1/40']
+        ],
+        'DisclosureDate' => '2020-01-28',
+        'License' => MSF_LICENSE,
+        'Platform' => 'unix',
+        'Arch' => ARCH_CMD,
+        'Privileged' => true,
+        'Targets' => [
+          ['OpenSMTPD < 6.6.1',
+           'MyBadChars' => "!\#$%&'*?`{|}~\r\n".chars]
```

Comment:
I kind of like this change. It tightens up targets.

Reply:
At the same time, it's kind of opposite the effect in https://github.com/rapid7/metasploit-framework/pull/12990/files#r388400580, but I think looking at the opening square brackets should be sufficient to visually differentiate targets.

Reply:
I thought I made this change and failed to indent the hash correctly (two spaces vs. one), but apparently RuboCop did that here. This was fixed in #13261. Nice!